JitsiConference.js

```diff
@@ -332,6 +332,16 @@ JitsiConference.prototype.addTrack = function (track) {
     this.room.addListener(XMPPEvents.SENDRECV_STREAMS_CHANGED,
         track.ssrcHandler);
 
+    // Report active device to statistics
+    var devices = RTC.getCurrentlyAvailableMediaDevices();
+    device = devices.find(function (d) {
```

Comment:
_setRealDeviceIdFromDeviceList is already public cause it is used by jitsi-meet, which I don't like and would prefer to remove that. Also I'm not sure when the deviceId is updated, for the above reason.